src/billing/useWorkspaceBillingProfile.test.ts

```diff
@@ -0,0 +1,86 @@
+import { Billing, BillingContract } from 'src/libs/ajax/Billing';
+import { WorkspaceManagerResources, WorkspaceManagerResourcesContract } from 'src/libs/ajax/WorkspaceManagerResources';
+import { asMockedFn, mockNotifications, renderHookInActWithAppContexts } from 'src/testing/test-utils';
+
+import { useWorkspaceBillingProfile } from './useWorkspaceBillingProfile';
+
+jest.mock('src/libs/ajax/Billing', () => ({ Billing: jest.fn() }));
+jest.mock('src/libs/ajax/WorkspaceManagerResources', () => ({ WorkspaceManagerResources: jest.fn() }));
+
+describe('useWorkspaceBillingProfile', () => {
+  it('returns billing profile', async () => {
+    // Arrange
+    const billingProfile = {
```

Comment:
Can you add the type for the test billing profiles to make sure they match the interface?

I also wonder if it would make sense to define a test one in a common location, like we do for other test data objets.

Reply:
Added a central fixture in https://github.com/DataBiosphere/terra-ui/pull/4904/commits/7c439cebd7d968d44576e8efeb20ddec22f06758.